### PR TITLE
Feature/horroy Oriol Bover

### PR DESCRIPTION
**Pull request by boversauros**

### CODE REVIEW

File: staff/oriol-bover/horroy/scripts/horroy.js [hunk 1]
```diff
@@ -0,0 +1,328 @@
+function Horroy() {
+    this.length = arguments.length;
+
```
Comment (maryammdot) [inline staff/oriol-bover/horroy/scripts/horroy.js:3]: You can control when the user enters an integer, it means you have to create an empty Horroy with that length. 🤓

File: staff/oriol-bover/horroy/scripts/horroy.js [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+ * 
+ * From
+ * 
+ * Creates a new shallow-copied horroy from another horroy
+ * 
+ * @param {*} value - 
+ * 
+ */
+
+Horroy.from = function (value) {
```
Comment (maryammdot) [inline staff/oriol-bover/horroy/scripts/horroy.js:43]: You should validate the parameter value

File: staff/oriol-bover/horroy/scripts/horroy.js [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+};
+
+Horroy.prototype.forEach = function (callback) {
+    for (var i = 0; i < this.length; i++)
+        callback(this[i]);
+};
+
+
+Horroy.prototype.fill = function (value, start, end) {
+    if (arguments.length > 4) throw Error('too many arguments');
```
Comment (maryammdot) [inline staff/oriol-bover/horroy/scripts/horroy.js:75]: You should also validate each parameter type

Comment (boversauros) [reply staff/oriol-bover/horroy/scripts/horroy.js:75]: Done! thanks

File: staff/oriol-bover/horroy/scripts/horroy.js (hunk 2 above)
Comment (boversauros) [reply staff/oriol-bover/horroy/scripts/horroy.js:43]: Done Appreciate it

**Pull request merged**